matchbox_socket/src/webrtc_socket/socket.rs

```diff
@@ -430,6 +443,23 @@ impl WebRtcSocket {
 }
 
 impl<C: ChannelPlurality> WebRtcSocket<C> {
+    /// Close this socket, disconnecting all channels.
+    pub fn close(mut self) {
+        self.channels
+            .iter_mut()
+            .filter_map(Option::take)
+            .for_each(|mut c| c.close());
+    }
+
+    /// Returns whether this socket is closed; this is considered closed as soon as any channel (not
+    /// taken) is closed.
+    pub fn is_closed(&self) -> bool {
```

Comment:
Not sure how I feel about `is_closed()` being true if any channel is closed. Perhaps we should implement `any_closed` and `all_closed` for any `ChannelPlurality` and `is_closed` for `SingleChannel`

Reply:
👍🏻 agree with this.